gitgud/__main__.py

```diff
@@ -241,7 +241,7 @@ def handle_load(self, args):
                     print("Challenge \"{}\" does not exist".format(args.challenge_name))
                     print("To view challenges/levels, use git gud challenges or git gud levels")
             else:
-                challenge = next(iter(level.challenges.values()))
+                challenge = next(iter(level.challenges))
                 challenge.setup(self.file_operator)
```

Comment:
We can make this one line similar to above: `level[0].setup(self.file_operator)`